src/app/components/homePage/shared/eventPortal.less

```diff
@@ -78,6 +78,7 @@
       justify-content: flex-start;
       border: solid;
       overflow: auto;
+      max-height: 300px;
```

Comment:
I did this because without it safari shows all jobs at once, there is not overflow